### PR TITLE
Update to match standard Meteor login and Account token storage

### PR DESCRIPTION
**Pull request by jazeee**
Meteor Accounts package uses hashedTokens instead of tokens. I updated the code to
match Accounts package.
Fix for https://github.com/kahmali/meteor-restivus/issues/79

### CODE REVIEW

**Comment by kahmali** [comment]
I probably won't have time to review this until tonight. A few quick things that I noticed after a quick glance (nothing code related yet):

1. We need to update the [change log](https://github.com/kahmali/meteor-restivus/blob/devel/CHANGELOG.md) with all relevant changes (it can go under an "Unreleased" section with no date)
1. We should add a fairly obvious warning at the top of the docs to let users know that their tokens may suddenly be invalid, and clients will be forced to reauthenticate

File: package.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'nimble:restivus',
   summary: 'Create authenticated REST APIs in Meteor 0.9+. Setup CRUD endpoints for Collections.',
-  version: '0.6.6',
+  version: '0.6.7',
```
Comment (kahmali) [inline package.js]: I'll bump the release in a separate commit. You can remove this.

Comment (jazeee) [inline package.js]: Ok, I made this update, and the changelog update.
I am learning git, so, I guess I will create the pull request. Hopefully it is not a duplicate.

**Comment by kahmali** [comment]
Any commits you push to the devel branch will automatically be included in this PR. What you need to do is use an interactive rebase to reorder those commits so that the delay commit is at the end, then create a branch for that, and a separate branch for everything on this PR, with the HEAD moved back a commit. You should make a new PR from that feature branch (and ALWAYS use a feature branch when making any PRs anywhere, from here on out). Then you'll want to squash all the commits on the PR branch together. Check out my [Contributing Guidelines](https://github.com/kahmali/meteor-restivus/blob/devel/CONTRIBUTING.md) for more information.

**Comment by kahmali** [comment]
Actually, what I suggested there would leave your delay commit stranded, so don't follow that exactly. I don't have time to research it right now so you'll have to look up the best way to separate that commit and make it's parent the commit before the PR commits. That way any further rebasing you do doesn't leave it stranded. I'm in a rush so I'm not even sure what I just said makes sense.

**Comment by jazeee** [comment]
Since it is a small change, I will look at reverting the code change and then doing it right.
Thanks.

File: lib/route.coffee [hunk 2]
```diff
@@ -207,8 +208,15 @@ class @Route
     # Send response
     endpointContext.response.writeHead statusCode, headers
     endpointContext.response.write body
-    endpointContext.response.end()
-
+    switch statusCode
```
Comment (jazeee) [inline lib/route.coffee]: I will extract this change into a separate branch and pull request.

**Comment by jazeee** [comment]
I created a separate branch, but since you don't have the other changes, I cannot create a pull request against your code yet.
For now, I have it in my branch "fix-issue-81".
https://github.com/jazeee/meteor-restivus/pull/1

**Comment by jazeee** [comment]
I branched and created a new pull request, which should be identical to this request, except with squashed commits.

https://github.com/kahmali/meteor-restivus/pull/82

**Pull request closed without merging**